tests/components/demo/test_humidifier.py

```diff
@@ -106,13 +105,13 @@ async def test_set_hold_mode_eco(hass):
     await hass.services.async_call(
         DOMAIN,
         SERVICE_SET_MODE,
-        {ATTR_MODE: MODE_ECO, ATTR_ENTITY_ID: ENTITY_HYGROSTAT},
+        {ATTR_MODE: "eco", ATTR_ENTITY_ID: ENTITY_HYGROSTAT},
```

Comment:
Using a string here as MODE_ECO is only used in tests, and is not imported into `homeassistant.components.humidifier` root.
See https://github.com/home-assistant/core/pull/78181